.github/workflows/build.yml

```diff
@@ -175,6 +179,38 @@ jobs:
           EXCLUDE_PATTERN: ${{ matrix.exclude-pattern || ' ' }}
           DEFAULT_FEATURE_FLAG_STATE: ${{ matrix.feature-flags }}
 
+      - name: Read flakey test results
+        id: set_flaky_test_results_var
+        run: |
+          input_file="rspec-retry-flakey-tests.txt"
+          while read line
+          do
+            file_text=$line
```

Comment:
Does this (over)write or append to the variable? We only seem to get the last line in the comment body...